src/dotnet/Program.cs

```diff
@@ -134,6 +135,12 @@ private static void PrintVersionInfo()
         {
             HelpCommand.PrintVersionHeader();
 
+            var commitSha = GetCommitSha();
+            Reporter.Output.WriteLine();
+            Reporter.Output.WriteLine("Product Information:");
+            Reporter.Output.WriteLine($" Version:     {HelpCommand.ProductVersion}");
+            Reporter.Output.WriteLine($" Commit Sha:  {commitSha.Substring(0, 10)}");
```

Comment:
this will cause a NullReferenceException when commitSha is null (e.g. when the file doesn't exist).